libs/ngx-dsfr/alert/src/lib/alert.component.html

```diff
@@ -1,8 +1,13 @@
 <div *ngIf="!isClosed" [class]="classes" [attr.role]="hasRole ? 'alert' : null">
   <h3 *ngIf="title" class="fr-alert__title">{{title}}</h3>
-  <p *ngIf="description">{{description}}</p>
-  <button *ngIf="hasCloseButton" class="fr-btn--close fr-btn" title="Masquer le message"
-          (click)="isClosed = true">
+  <p *ngIf="displayDescription" #description>
```

Comment:
So... I've given it a lot of thought and imagined several ways to do differently but it turns out that we can actually simplify the Component by not hiding this `<p>` tag even when there's no content projected.
Turns out that visually, it makes no difference ! 😳
The `<p>` tag is just a zero height block, with a 4px bottom margin that actually sits well inside the 4px bottom padding of the parent `<div>`... so whether it's there or not, nobody cares 🤷🏻‍♂️

Therefore, since it doesn't change anything for the user and it will prevent us from having to deal with change detection, I suggest to get rid of `*ngIf="displayDescription"`